scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -158,13 +158,13 @@ fi
 
 if [ "${CREATE_SCRIPT}" = "true" ]
 then
- echo -e "#!/bin/bash\ncd \`dirname \${0}\`\ncd workspaces/${WORKSPACE}\nsource ${DEVON_IDE_HOME}/scripts/devon eclipse" > eclipse-$WORKSPACE
- chmod a+x eclipse-$WORKSPACE
+ echo -e "#!/bin/bash\ncd \$(dirname \${0})\ncd workspaces/${WORKSPACE}\nsource ${DEVON_IDE_HOME}/scripts/devon eclipse" > eclipse-"$WORKSPACE"
+ chmod a+x eclipse-"$WORKSPACE"
  echo "Created script eclipse-$WORKSPACE"
 fi
 
 if [ "${run}" != "false" ]
 then
   echo "launching Eclipse..."
-  eclipse -clean -data "${WORKSPACE_PATH}" -keyring "~/.eclipse/.keyring" -vm "${JAVA_HOME}/bin/java" -showlocation ${WORKSPACE} -vmargs ${ECLIPSE_VMARGS} &
+  eclipse -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/java" -showlocation "${WORKSPACE}" -vmargs "${ECLIPSE_VMARGS}" &
```

Comment:
good catch. You fixed a bug here. `~` is not resolved if quoted...

Reply:
I know, that the reason for the log message  (another ShellCheck finding) "Tilde does not expand in quotes" as listed on top of PR.